specs/vm/instruction_set.md

```diff
@@ -1417,6 +1418,22 @@ This modifies the `balanceRoot` field of the appropriate output.
 `messageID` is added to the `OutputMessage` Merkle tree as part of block header.
 TODO: document output messages merkle tree construction and maintenance and link here
 
+### SCWQ: State clear sequential 32 byte slots
+
+|             |                                                                               |
+|-------------|-------------------------------------------------------------------------------|
+| Description | A sequential series of 32 bytes is cleared from the current contract's state. |
+| Operation   | ```STATE[MEM[$rA, 32 * rB]] = None;```                                        |
+| Syntax      | `scwq $rA, $rB`                                                               |
+| Encoding    | `0x00 rA rB - -`                                                              |
+| Notes       |                                                                               |
+
+Panic if:
+
+- `$rA + 32 * rB` overflows
+- `$rA + 32 * rB > VM_MAX_RAM`
+- `$fp == 0` (in the script context)
+
```

Comment:
Should we also set `rB` here to indicate whether the initial key actually existed?

Reply:
Unlike x86, we don't re-use registers anywhere in the instruction set. So if we want to return something, we should use a third register.

Reply:
yeah I was assuming we'd shift the current use of rB into rC to be more consistent with the other recent changes

Reply:
fixed